### DISCUSSION TITLE
Would anybody be so awesome to tell me how to target this file?

### DISCUSSION THREAD

**Discussion by Elindo586** [vercel/next.js#38380] [Help] [answered]
I need to target the parts-parker.json  from the  parker.js file 

Can I do something like?

`import db from ".../utils/parts-parker" `      <-  this is getting me an error 

or

` import db from "/utils/parts-parker"`  <- this doesn't get errors, but the data is just blanked, nothing, nada.. 

![202020](https://user-images.githubusercontent.com/93617433/177601648-f33738f0-232c-4305-b4e4-ac54d75b4d92.PNG)

**Comment by icyJoseph** [comment, marked as answer]
Hi,

`../../utils/parts-parker` should work right? Goes up once to invent, then once again to `pages(?)` and from there down to `utils`,  and there it finds the `JSON` file.

**Comment by Elindo586** [comment]
Hmm... yes... that worked.... it did let me finished the typing and seems like it is targeting the right document...

Say..... is there a limit on how many or how big a json file is? 

Now the page is loading without errors, but still not pulling the data....

**Comment by Elindo586** [comment]
Oh I got it...

Needed to fix something form the json file for it to work...   thanks a million for you all your awesome help!
